src/main/java/fishron/AddCommand.java

```diff
@@ -0,0 +1,35 @@
+package fishron;
```

Comment:
You can consider creating a separate package for the Command class and its subclasses, instead of packing everything under the "fishron" package. Will be helpful as the project gets bigger :)

Reply:
yes, since you created a lot of classes, it may be a bit difficult for the readers to understand the structure of your program.